rustainers/src/runner/runner_network.rs

```diff
@@ -0,0 +1,60 @@
+use serde::{Deserialize, Serialize};
+
+/// The network driver
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
+pub enum Driver {
```

Comment:
🔨 **todo**: There is a similar type `crate::container::network::Network`
Null -> None
Ipvlan, & macvlan -> Custom ?
Overlay ? maybe custom, maybe this is a container ?